packages/embed/src/factories/create-popover/create-popover.ts

```diff
@@ -97,6 +97,48 @@ const buildTooltip = (message: string, hide: () => void) => {
   return container
 }
 
+const LOCAL_STORAGE_NOTIFICATION_DATA = 'tfNotificationData'
+
+type LocalStorageCUIData = {
+  formId: string
+  expireDate: number
+}
+const today = new Date().getTime()
+
+const cuiLocaslStorageDate = typeof window !== 'undefined' && localStorage.getItem(LOCAL_STORAGE_NOTIFICATION_DATA)
+
+const getLocalStorageData = () => cuiLocaslStorageDate && JSON.parse(cuiLocaslStorageDate)
```

Comment:
I would not access the variable from outer scope in a function, but rather read the value inside the function. I would also name it "data" not "date" (it is an object with form id and date).
```suggestion
const getLocalStorageData = () => {
  const data = typeof window !== 'undefined' && localStorage.getItem(LOCAL_STORAGE_NOTIFICATION_DATA)
  return data && JSON.parse(data)
}
```

Reply:
typo!